spec.emu

```diff
@@ -239,6 +243,22 @@ contributors: Google, Ecma International
         The value of the [[RelevantExtensionKeys]] internal slot is &laquo;  &raquo;.
       </p>
 
+      <p>
+        The value of the [[LocaleData]] internal slot is implementation defined within the constraints described in <emu-xref href="#sec-internal-slots"></emu-xref> and the following additional constraints:
+      </p>
+
+      <ul>
+        <li>
+        [[LocaleData]].[[&lt;_locale_&gt;]] must have a types field for all locale values _locale_.
+        The value of this field must be a record, which must have fields with the names of the six display name types:
+        `"language"`, `"region"`, `"script"`, `"currency"`, `"dateField"`, and `"dateSymbol"`.
+        Each of these fields in turn must be a record, which must have fields with the names of the three display name styles:
+        `"narrow"`, `"short"`, and `"long"`.
+        The value of these fields in turn must be a record.
+        The values corresponding to the property keys are String values.
```

Comment:
Please describe in more detail what the fields in this record represent.  I assume they are the identifiers that you pass to the `.of` method?  Do you have language for what you call those identifiers?

Reply:
done